server/routes/user.js

```diff
@@ -1,10 +1,10 @@
 import express from 'express';
 import UserController from '../controllers/userController';
-
+import InputValidator from '../middlewares/inputValidator';
 
 const userRoute = express();
 
```

Comment:
I think you can use destructuring here. Something like 
```
const { validateUser, validateLogin } = InputValidator;
const { signUp, login } = UserController;
```
and then 
userRoute.post('/', validateUser, signup).

What do you think.

Reply:
Nice. Good idea. I will do that now. Thanks.

Reply:
> Nice. Good idea. I will do that now. Thanks.

Good job